lib/practice_exercises.rb

```diff
@@ -1,13 +1,32 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n) - Need to go through each index of the array to compare its values.
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  list.each do |i|
+    if list[i] == list[i+1]
+      list.delete_at(i)
```

Comment:
because `delete_at` shifts each element 1 step to the left with each iteration, it runs at O(n).  So your total method is O(n<sup>2</sup>).

Otherwise this works well.